1.0/en/0x10-C02-User-Input-Validation.md

```diff
@@ -62,6 +62,44 @@ Even syntactically valid prompts may request disallowed content (illicit instruc
 
 ---
 
+## C2.5 Input Rate Limiting & Abuse Prevention
+
+Prevent abuse, resource exhaustion, and automated attacks by limiting input rates and detecting anomalous usage patterns.
+
+| # | Description | Level | Role |
+|:--------:|---------------------------------------------------------------------------------------------------------------------|:---:|:---:|
+| **2.5.1** | **Verify that** per-user, per-IP, and per-API-key rate limits are enforced for all input endpoints. | 1 | D/V |
+| **2.5.2** | **Verify that** burst and sustained rate limits are tuned to prevent DoS and brute-force attacks. | 2 | D/V |
+| **2.5.3** | **Verify that** anomalous usage patterns (e.g., rapid-fire requests, input flooding) trigger automated blocks or escalations. | 2 | D/V |
```

Comment:
See current content in [Chapter 12](https://github.com/OWASP/AISVS/blob/main/1.0/en/0x10-C12-Monitoring-and-Logging.md) regarding suggested 2.5.3 and 2.5.4.

However I think the 2.5.1 & 2.5.2 are good additions to this control :+1: